### PR TITLE
update changelog for v1.11.0 release

### PR DESCRIPTION
**Pull request by TheConcierge**
This captures the rest of the changes since the `v1.10.0` release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,13 @@
 ## 1.11.0
 
+Enhancements:
+
+- Adds `WithAllowsPooling` option
+- Adds `WithURL` option
```
Comment (jrobsonchase) [inline CHANGELOG.md:6]: Might be nice to have a little blurb about what each of these actually do. 

They don't actually have godocs at the moment, which is also a problem, but probably not one for this change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+
 Changes:
 
-- Adds `WithTrafficPolicy` function, replacing `WithPolicyString`
+- Replaces `WithPolicyString` option with `WithTrafficPolicy`
```
Comment (jrobsonchase) [inline CHANGELOG.md:10]: Can we change the wording of this to indicate that `WithPolicyString` is just _deprecated_ and not replaced entirely? Was about to come in hot with the "no breaking changes without a major version" hammer, but then discovered that that isn't actually the case :sweat_smile:

File: CHANGELOG.md (hunk 1 above)
Comment (TheConcierge) [reply CHANGELOG.md:6]: this would be nice! however, i have no idea what they do!

File: CHANGELOG.md (hunk 2 above)
Comment (TheConcierge) [reply CHANGELOG.md:10]: yeah you're right, this wording isn't so good. I'll probably move `WithTrafficPolicy` to enhancements and leave a deprecation message for `WithPolicyString`

File: CHANGELOG.md (hunk 1 above)
Comment (TheConcierge) [reply CHANGELOG.md:6]: I reached out to the endpoints project to see if anybody had anything additional, but got no response. I may need to nudge them again

**Review by jrobsonchase — APPROVED** [review]

**Pull request merged**